### DISCUSSION TITLE
I get an error when I try to use , large-V3

### DISCUSSION THREAD

**Discussion by Malestory** [openai/whisper#1788] [Q&A]
Error transcribing file on line Invalid model size 'large-v3', expected one of: tiny.en, tiny, base.en, base, small.en, small, medium.en, medium, large-v1, large-v2, large

The error above appears.
But,
I have the large-v2 model installed
Why is this like this?

**Comment by glangford** [comment]
Have you updated whisper to the latest release?

> The large-v3 model is available in openai-whisper==20231106 and after.

**Reply by Malestory** [reply]
yes. Updated.
So I was able to choose V3.

**Reply by jasonsu123** [reply]
How to update?

I had the same problems
available models = ['tiny.en', 'tiny', 'base.en', 'base', 'small.en', 'small', 'medium.en', 'medium', 'large-v1', 'large-v2', 'large']

Thanks

**Comment by 93041025** [comment]
same issue here. have you solved the problem?

"Error transcribing file on line Invalid model size 'large-v3', expected one of: tiny.en, tiny, base.en, base, small.en, small, medium.en, medium, large-v1, large-v2, large
/home/qq/miniconda3/envs/whisper/lib/python3.9/site-packages/torch/cuda/memory.py:303: FutureWarning: torch.cuda.reset_max_memory_allocated now calls torch.cuda.reset_peak_memory_stats, which resets /all/ peak memory stats.
  warnings.warn(
Error transcribing file on line Invalid model size 'large-v3', expected one of: tiny.en, tiny, base.en, base, small.en, small, medium.en, medium, large-v1, large-v2, larg"
